pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/ref_identifying-module-types.adoc

```diff
@@ -0,0 +1,46 @@
+[id='identifying-module-types_{context}']
+= Identifying {ProductShortName} module types
+:pantheon-module-type: REFERENCE
+
+In Pantheon 2, each module must be identified by type: concept, procedure, or reference.
+
+Module types are described in https://redhat-documentation.github.io/modular-docs/#writing-mod-docs[Types of modules in Red Hat documentation].
+
+Identification is done by using either one or both of the following:
+
+* The correct attribute after the module's title
+* The correct prefix in the module's name
+
+Use of the correct attribute (preferred method)::
+
+One of the following attributes should be entered in the line immediately following the module's title:
+
+* `:pantheon-module-type: CONCEPT`
+* `:pantheon-module-type: PROCEDURE`
+* `:pantheon-module-type: REFERENCE`
+
+====
+.Example
+
+`= Identifying {ProductShortName} module types :pantheon-module-type: REFERENCE`
+
+====
+
+Use of the correct prefix::
+
+Module titles should start with the appropriate choice among the following
+
+* `con_`
+* `proc_`
+* `ref_`
+
+====
+.Example
+
+`ref_identifying-module-types.adoc`
+
+====
+
+How Pantheon 2 checks identifiers::
+
+Pantheon 2 first checks for the module's attribute. Any value than those specified is ignored. If there is no valid attribute, Pantheon 2 then checks the module's prefix. If both a valid attribute and prefix exist, Pantheon 2 defaults to the attribute.
```

Comment:
Any value *other* than...

(Missing *other*)

I know what you mean by "defaults to", but I'm not sure that's the correct usage...would like to hear someone else's opinion. I would say "{product-name} uses the attribute"

Reply:
Fixed; added missing word and went with "uses".